protocol/claim_monero.go

```diff
@@ -20,6 +21,11 @@ var (
 	log = logging.Logger("protocol")
 )
 
+// SwapManager is the interface for the swap manager.
```

Comment:
I'm not sure the comment says anything that isn't obvious from the name. Would this phrase be correct?
```
// SwapManager is the subset of the swap.Manager interface needed by ClaimMonero
```